libr/core/cmd_meta.c

```diff
@@ -575,6 +576,19 @@ static int cmd_meta_hsdmf(RCore *core, const char *input) {
 			free (comment);
 		}
 		break;
+	case '.':
+		str = r_meta_get_string (core->anal, type, addr);
+		if (str) {
+			if (type == 's') {
+				char *esc_str = r_str_escape_latin1 (str, false, true);
+				r_cons_printf ("\"%s\"\n", esc_str);
```

Comment:
check for null

Reply:
I always thought that r2 has undefined behavior if it runs out of memory, but see #8680 for a possible solution.